sa/migrations.sh

```diff
@@ -116,26 +124,27 @@ Usage:
   
   Boulder DB Migrations CLI
 
-  Helper for listing, promoting, and demoting Boulder schema files
+  Helper for listing, promoting, and demoting migration files
 
   ./$(basename "${0}") [OPTION]...
-
-  -l, --list-next           Lists schema files present in sa/_db-next
-  -c, --list-current        Lists schema files promoted from sa/_db-next to sa/_db 
-  -p, --promote             Select and promote a schema from sa/_db-next to sa/_db
-  -d, --demote              Select and demote a schema from sa/_db to sa/_db-next
+  -b  --db                  Name of the database, this is required (e.g. boulder_sa or incidents_sa)
+  -n, --list-next           Lists migration files present in sa/_db-next/<db>
+  -c, --list-current        Lists migration files promoted from sa/_db-next/<db> to sa/_db/<db> 
+  -p, --promote             Select and promote a migration from sa/_db-next/<db> to sa/_db/<db>
+  -d, --demote              Select and demote a migration from sa/_db/<db> to sa/_db-next/<db>
```

Comment:
wow I have never used any of these four modes

Reply:
Well, they are super neat for their intended purpose :)